### DISCUSSION TITLE
Xamarin and iOS 18 Beta - I know I know is xamarin but Maui....

### DISCUSSION THREAD

**Discussion by developer9969** [dotnet/maui#23263] [General]
Hi everyone,
We have been porting our apps to Maui and some are not prod ready due to bugs not being fixed by Microsoft, our clients rejected the app till these bugs are fixed.. expecially some odd keyboard ones.

I am about to try iOS 18 beta on xamarin in the hope that somehow works.. 

Does it work for you?

thanks

**Comment by giuseppenovielli** [comment]
@developer9969 Do you have tested Xamarin.Forms with iOS 18 Beta? It Works?

**Reply by developer9969** [reply]
Hi - it worked for me

**Reply by giuseppenovielli** [reply]
Ok, but did you try to compile Xamarin.Forms app with Xcode 16 beta? It Works?

**Reply by developer9969** [reply]
nope - as far I am concerned I just tried that an already built/deployed app works with with the beta version installed on the phone. I did not try compiling  as we no longer aim to deploy xamarin

**Reply by giuseppenovielli** [reply]
Ok, Thanks!
